conformance/tests/gateway-with-attached-routes.go

```diff
@@ -125,7 +120,7 @@ var GatewayWithAttachedRoutes = suite.ConformanceTest{
 			hrouteNN := types.NamespacedName{Name: "http-route-4", Namespace: "gateway-conformance-infra"}
 			notAccepted := metav1.Condition{
 				Type:   string(v1.RouteConditionAccepted),
-				Status: metav1.ConditionTrue,
+				Status: metav1.ConditionFalse,
```

Comment:
is it conflicting with https://github.com/kubernetes-sigs/gateway-api/blob/faba2585346ef0806585f8ce2ee458e9cd70ec7b/conformance/tests/httproute-invalid-backendref-nonexistent.go#L48 ?

Reply:
Yeah. I don't see how an implementation can pass both of these tests.
We have [this resource that should be `Accepted`](https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/tests/httproute-invalid-backendref-nonexistent.yaml) as @sayboras mentioned that's essentially identical to [the resource in the test from this PR](https://github.com/kubernetes-sigs/gateway-api/blob/93da4e4621151e72d1ef3b4515611bc5d1d31adf/conformance/tests/gateway-with-attached-routes.yaml#L118-L132) that should NOT be `Accepted`.

Reply:
Addressed in